ui/mirage/factories/agent.js

```diff
@@ -8,20 +8,51 @@ const AGENT_STATUSES = ['alive', 'leaving', 'left', 'failed'];
 
 export default Factory.extend({
   id: i => (i / 100 >= 1 ? `${UUIDS[i]}-${i}` : UUIDS[i]),
-  name: () => `nomad@${faker.random.boolean() ? faker.internet.ip() : faker.internet.ipv6()}`,
+  config: makeConfig,
```

Comment:
Is the correct pattern here to have a function using the `make` prefix?

Reply:
Looking at similar factories (like [node](https://github.com/hashicorp/nomad/blob/main/ui/mirage/factories/node.js) or [task-group](https://github.com/hashicorp/nomad/blob/main/ui/mirage/factories/task-group.js)) it seems like the `make` pattern is used when the resulting object is dynamic, which doesn't seem to be the case here since we are hard coding a version.

So I think we can just have this as:

```js
export default Factory.extend({
  id: i => (i / 100 >= 1 ? `${UUIDS[i]}-${i}` : UUIDS[i]),

  config: {
    Version: {
      Version: '1.1.0',
      VersionMetadata: 'ent',
      VersionPrerelease: 'dev',
    },
  },
// ...
```

Reply:
Great call out.